src/runtime/importhook.cs

```diff
@@ -99,8 +83,12 @@ internal static unsafe void Initialize()
 
             Runtime.PyDict_Update(mod_dict, clr_dict);
             BorrowedReference dict = Runtime.PyImport_GetModuleDict();
-            Runtime.PyDict_SetItemString(dict, "CLR", ClrModuleReference);
-            Runtime.PyDict_SetItemString(dict, "clr", ClrModuleReference);
+            Runtime.PyDict_SetItemString(dict.DangerousGetAddress(), "CLR", py_clr_module);
+            Runtime.PyDict_SetItemString(dict.DangerousGetAddress(), "clr", py_clr_module);
```

Comment:
why are you switching to DangerousGetAddress here?

Reply:
I think that was an aritfact from rebasing